test/per_file_coverage.sh

```diff
@@ -66,7 +65,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/wasm_runtime/wavm:0.0" # Noe enabled in coverage build
 "source/extensions/watchdog:85.7" # Death tests within extensions
 "source/extensions/watchdog/profile_action:85.7"
-"source/server:94.5"
+"source/server:94.6"
```

Comment:
This is the other one I found to be flaky.  https://github.com/envoyproxy/envoy/pull/14336#issuecomment-751850442